### PR TITLE
HTML: test fieldset overflow rendering

### PR DESCRIPTION
**Pull request by zcorpan**
Bugs:
https://bugs.chromium.org/p/chromium/issues/detail?id=786475
https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/18694230/

### CODE REVIEW

**Review by annevk — APPROVED** [review]
@MatsPalmgren you might want to double check this.

File: html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+div {
+  background: red;
+  height: 2em;
+}
+
+</style>
+<p>There should be no red.</p>
+<fieldset>
+<legend></legend>
```
Comment (mstensho) [inline html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow.html]: Missing indentation.

File: html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow-ref.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+.f {
+  overflow: auto;
+  height: 1em;
+  margin-top: 2em;
+}
+.l {
+  height: 1em;
+  width: 5em;
+  line-height: 1;
+  display: table;
```
Comment (mstensho) [inline html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow-ref.html]: No need for this.

File: html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow-ref.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<style>
+.f, .l {
+  border: 1em solid;
+  margin: 0;
+  padding: 0;
+  background: lime;
+}
+
+.f {
+  overflow: auto;
```
Comment (mstensho) [inline html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow-ref.html]: No need for this.

File: html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow-ref.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML>
+<title>fieldset overflow</title>
+<style>
+.f, .l {
+  border: 1em solid;
+  margin: 0;
```
Comment (mstensho) [inline html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow-ref.html]: No need to specify margin / padding when only used on DIVs.

**Review by mstensho — APPROVED** [review]
Nits. :)

**Comment by zcorpan** [comment]
PTAL

File: html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow-ref.html [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  line-height: 1;
+  margin-top: -4em;
+  margin-left: 1em;
+}
+</style>
+<p>There should be no red.</p>
+<div class=fieldset>
+</div>
+<div class=legend></div>
+
```
Comment (mstensho) [inline html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-overflow-ref.html]: blank line

**Review by mstensho — APPROVED** [review]
lgtm

**Comment by gsnedders** [comment]
Is there any reason this hasn't landed?

**Comment by zcorpan** [comment]
Don't think so

**Pull request merged**